examples/react/start-mcp/src/routes/__root.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import {
+  HeadContent,
+  Outlet,
+  Scripts,
+  createRootRoute,
+} from '@tanstack/react-router'
+import { TanStackRouterDevtools } from '@tanstack/react-router-devtools'
+
+export const Route = createRootRoute({
+  head: () => ({
+    meta: [
+      {
+        charSet: 'utf-8',
+      },
+      {
+        name: 'viewport',
+        content: 'width=device-width, initial-scale=1',
+      },
+      {
+        title: 'TanStack Start MCP Starter',
+      },
+    ],
+  }),
+  component: RootComponent,
```

Comment:
please use shellComponent

see https://github.com/TanStack/router/blob/main/examples/react/start-basic/src/routes/__root.tsx#L62

(yes I know i need to document this :D )